common_features.mk

```diff
@@ -250,10 +251,10 @@ ifneq ($(strip $(BACKLIGHT_ENABLE)), no)
     SRC += $(QUANTUM_DIR)/backlight/backlight.c
     OPT_DEFS += -DBACKLIGHT_ENABLE
 
-    ifeq ($(strip $(BACKLIGHT_ENABLE)), software)
+    ifeq ($(strip $(BACKLIGHT_DRIVER)), software)
         SRC += $(QUANTUM_DIR)/backlight/backlight_soft.c
     else
-        ifeq ($(strip $(BACKLIGHT_ENABLE)), custom)
+        ifeq ($(strip $(BACKLIGHT_DRIVER)), custom)
```

Comment:
if this comparison necessary if it would error out before reaching this point?

Reply:
Due to backlight_avr still containing all implementations (pwm/timer/software/custom), we need a way to set the `BACKLIGHT_CUSTOM_DRIVER` define while still pulling some of the avr code to keep existing keyboards happy. Hopefully this should not be a long term bodge.